presto-docs/src/main/sphinx/sql/alter-table.rst

```diff
@@ -14,6 +14,7 @@ Synopsis
     ALTER TABLE [ IF EXISTS ] name ADD [ CONSTRAINT constraint_name ] { PRIMARY KEY | UNIQUE } ( { column_name [, ...] } ) [ { ENABLED | DISABLED } ] [ [ NOT ] RELY ] [ [ NOT ] ENFORCED } ]
     ALTER TABLE [ IF EXISTS ] name DROP CONSTRAINT [ IF EXISTS ] constraint_name
     ALTER TABLE [ IF EXISTS ] ALTER [ COLUMN ] column_name { SET | DROP } NOT NULL
+    ALTER TABLE [ IF EXISTS ] SET PROPERTIES (property_name=value)
```

Comment:
```suggestion
    ALTER TABLE [ IF EXISTS ] name SET PROPERTIES (property_name=value [, ...])
```

The properties may be more than 1, so is it better to write it in a list format?